kubernetes/resource_kubernetes_default_service_account.go

```diff
@@ -59,6 +60,25 @@ func resourceKubernetesDefaultServiceAccountCreate(ctx context.Context, d *schem
 		return diag.FromErr(err)
 	}
 
+	if v, ok := d.GetOkExists("automount_service_account_token"); ok {
```

Comment:
This condition is verifying whether the argument `automount_service_account_token` is in the terraform code and if so, it will be taken to apply. Let's remove the condition and let Terraform pick up a default value of the argument if it is not set. In this particular case, it will be `true` and that is matches with the default value on the Kubernetes end.

Reply:
Thanks for the review @arybolovlev 
Since the default value is `true`, `GetOkExists` will also get that default value but seems like we don't have a use case to use `GetOkExists` here.